streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignorTest.java

```diff
@@ -552,48 +536,60 @@ public void testAssignWithStandbyReplicas() throws Exception {
 
     @Test
     public void testOnAssignment() throws Exception {
-        TopicPartition t2p3 = new TopicPartition("topic2", 3);
+        configurePartitionAssignor(Collections.<String, Object>emptyMap());
+
+        final List<TaskId> activeTaskList = Utils.mkList(task0, task3);
+        final Map<TaskId, Set<TopicPartition>> activeTasks = new HashMap<>();
+        final Map<TaskId, Set<TopicPartition>> standbyTasks = new HashMap<>();
+        final Map<HostInfo, Set<TopicPartition>> hostState = Collections.singletonMap(
+                new HostInfo("localhost", 9090),
+                Utils.mkSet(t3p0, t3p3));
+        activeTasks.put(task0, Utils.mkSet(t3p0));
+        activeTasks.put(task3, Utils.mkSet(t3p3));
+        standbyTasks.put(task1, Utils.mkSet(t3p1));
+        standbyTasks.put(task2, Utils.mkSet(t3p2));
+
+        final AssignmentInfo info = new AssignmentInfo(activeTaskList, standbyTasks, hostState);
+        final PartitionAssignor.Assignment assignment = new PartitionAssignor.Assignment(Utils.mkList(t3p0, t3p3), info.encode());
+
+        Capture<Map<TaskId, Set<TopicPartition>>> capturedActiveTasks = EasyMock.newCapture();
+        Capture<Map<TaskId, Set<TopicPartition>>> capturedStandbyTasks = EasyMock.newCapture();
+        Capture<Map<HostInfo, Set<TopicPartition>>> capturedHostState = EasyMock.newCapture();
+        Capture<Cluster> capturedCluster = EasyMock.newCapture();
+        taskManager.setPartitionsByHostState(EasyMock.capture(capturedHostState));
```

Comment:
Do we need the `Capture` here? Can't we just pass in the excepted `Set` etc and then `verify` at the end?